plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -259,7 +259,7 @@ public TableInfo getCachedTable(Duration viewExpiration, TableInfo remoteTableId
      */
     public String getParentProjectId()
```

Comment:
Please update the method name. e.g. `getQuotaProjectId`

Reply:
I updated method so now it does not always return `QuotaProjectId`, but projectId dependent on configuration. Name is aligned with CreateReadSessionRequest API:
https://github.com/trinodb/trino/blob/e13da7386edd1c55e984971664c6b33deb583172/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/ReadSessionCreator.java#L103-L104